test/utils/run_tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+PATH=$PATH:node_modules/.bin
+
+runTest ( ) {
+    if [ "$1" = "sqlite" ]
```

Comment:
Since the shebang is set on `bash`, let's use *bash-esque* constructs and turn this into:

```sh
if [[ "$1" == "sqlite" ]]; then
  ...
```

Elsewhere in the file too :)